runtime/JavaScript/package.json

```diff
@@ -1,12 +1,26 @@
 {
   "name": "antlr4",
   "version": "4.12.0",
-  "type": "module",
```

Comment:
are you proposing to no longer build a module ?

Reply:
No, the `type` module is now only in the `es` directory. Previously this library wasn't actually exporting a module (webpack doesn't output esm modules) but said it was. That was part the motivation of this PR, this kind of stuff breaks builds for other libraries that are consuming antlr4.

Now this package will export both commonjs & esm and support both.